### PR TITLE
toggle button added for time selection and date selection in date picker widget, related to #3576

### LINKED ISSUE #3576: Add toggle button to enable widget properties
### What is the expected behaviour?

Add buttons for “Enable time selection” & “Enable date selection”, so that users can easily toggle true and false values.

### What is the current behaviour?

Toggle button missing in “Enable time selection” & “Enable date selection”  properties.

### How to reproduce the issue?

1. Go to this [link](https://app.tooljet.com/)
2. Now click on "create new application" 
3. Now drag and drop the date-picker widget from the widgets inventory
4. Inspect the widget and scroll down to its properties panel on the right
5. Now under sections “Enable time selection” & “Enable date selection” we have to add toggle buttons 

### Screenshots or Screencast

Screenshot 2022-07-11 at 4 12 57 PM

### Any possible solutions?

Refer to other widgets by inspecting them to learn about the UI.

### PR DESCRIPTION
**Pull request by kaggrwal**
toggle button added for time selection and date selection in date picker widget, 

fixes #3576

### CODE REVIEW

**Comment by Navaneeth-pk** [comment]
@kaggrwal please add proper title for the PR.

**Comment by kaggrwal** [comment]
@Navaneeth-pk I have edited the title along with the original issue.

**Comment by kaggrwal** [comment]
@Navaneeth-pk Can you please merge it  ?

**Comment by withshubh** [comment]
> @Navaneeth-pk Can you please merge it ?

Hey - Please wait for the PR to be reviewed. :)

File: frontend/src/Editor/WidgetManager/widgetConfig.js [hunk 1]
```diff
@@ -1142,20 +1142,8 @@ export const widgets = [
...
-        },
-      },
-      enableDate: {
-        type: 'code',
-        displayName: 'Enable date selection?',
-        validation: {
-          schema: { type: 'boolean' },
-        },
-      },
+      enableTime: { type: 'toggle', displayName: 'Enable time selection?' },
```
Comment (arpitnath) [inline frontend/src/Editor/WidgetManager/widgetConfig.js:1145]:
```suggestion
            enableTime: {
        type: 'toggle',
        displayName: 'Enable time selection?',
        validation: {
          schema: { type: 'boolean' },
          defaultValue: false,
        },
      },
      enableDate: {
        type: 'toggle',
        displayName: 'Enable date selection?',
        validation: {
          schema: { type: 'boolean' },
          defaultValue: true,
        },
      },
```

Comment (arpitnath) [reply frontend/src/Editor/WidgetManager/widgetConfig.js:1145]: Please can you add the validations for the widget property?

**Review by arpitnath — CHANGES_REQUESTED** [review]

File: frontend/src/Editor/WidgetManager/widgetConfig.js (hunk 1 above)
Comment (kaggrwal) [reply frontend/src/Editor/WidgetManager/widgetConfig.js]: Vaidations added

**Comment by kaggrwal** [comment]
@arpitnath can you please review and merge it ?

**Review by arpitnath — APPROVED** [review]

**Comment by withshubh** [comment]
Hey @kaggrwal  
Thanks for contributing to ToolJet. :raised_hands: 
Please submit the form to claim your Hacktoberfest swag: https://airtable.com/shr1qR6xhqsWO7LD7

**Pull request merged**